packages/communication-react/CHANGELOG.stable.md

```diff
@@ -1,9 +1,49 @@
 # Change Log - @azure/communication-react
 
-This log was last generated on Fri, 06 Oct 2023 19:59:45 GMT and should not be manually modified.
+This log was last generated on Mon, 06 Nov 2023 16:44:11 GMT and should not be manually modified.
 
 <!-- Start content -->
 
+## [1.10.0](https://github.com/azure/communication-ui-library/tree/@azure/communication-react_v1.10.0)
+
+Mon, 06 Nov 2023 16:44:11 GMT 
+[Compare changes](https://github.com/azure/communication-ui-library/compare/@azure/communication-react_v1.10.0-beta.1...@azure/communication-react_v1.10.0)
+
```

Comment:
Should have a sub-heading here for precedence like:
`### Custom Button Injection - General Availability`

Reply:
changed. Will only affect what we merge back into main